gateway-ha/src/test/java/io/trino/gateway/ha/persistence/TestDatabaseMigrationsOracle.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.gateway.ha.persistence;
+
+import com.google.common.collect.ImmutableList;
+import org.jdbi.v3.core.Handle;
+import org.testcontainers.containers.JdbcDatabaseContainer;
+import org.testcontainers.containers.OracleContainer;
+
+import java.util.List;
+
+public class TestDatabaseMigrationsOracle
+        extends BaseTestDatabaseMigrations
+{
+    @Override
+    protected final JdbcDatabaseContainer<?> startContainer()
+    {
+        JdbcDatabaseContainer<?> container = new OracleContainer("gvenzl/oracle-xe:18.4.0-slim");
```

Comment:
We should document the tested version of Oracle. I don't see anything that would prevent using older versions - the SQL is very standard and no varchars are longer than 256. But if it isn't documented, we'll get questions

Reply:
And we should also update the docs to include Oracle as a backend option

Reply:
support for `FETCH` in oracle was introduced in 12c so at least 12c will be required. I will add this to the docs.